machine-learning/solutions/seismic_salt_detection/trainer/model.py

```diff
@@ -127,14 +116,15 @@ def mean_absolute_error(labels, predictions):
         h_units.append(math.ceil(layer * lfrac))
         layer = h_units[-1]
 
-    estimator = tf.estimator.DNNRegressor(
+    estimator = tf.estimator.DNNClassifier(
```

Comment:
There are still places in this file that refer to the estimator as a regressor.

Reply:
will fix